runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -385,59 +344,23 @@ else if (poisonVarp >= VENOM_VALUE_CUTOFF)
 				nextOverloadRefreshTick = tickCount + OVERLOAD_TICK_LENGTH;
 			}
 
-			if (overloadVarb <= 0)
-			{
-				nextOverloadRefreshTick = -1;
```

Comment:
The `timers: Fix overload duration after timer is removed` commit is actually fixing a bug introduced here, and not from 62fc017d9629b33ce9d48ec75fbd6abbbacb993f. Previous code would assign nextOverloadRefreshTick = -1 on overloadVarb <= 0

Reply:
Agreed, I've `fixup`'d this commit into the var consolidation change, now ace6d2e3ba96aa43d5c7be55a15bdade91c16448